backend/src/v5/processors/teamspaces/projects/models/containers.js

```diff
@@ -15,15 +15,32 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+const { addModel, deleteModel, getModelList } = require('./commons/modelList');
 const { appendFavourites, deleteFavourites } = require('./commons/favourites');
 const { getContainerById, getContainers } = require('../../../../models/modelSettings');
 const { getLatestRevision, getRevisionCount, getRevisions, updateRevisionStatus } = require('../../../../models/revisions');
 const Groups = require('./commons/groups');
-const { getModelList } = require('./commons/modelList');
 const { getProjectById } = require('../../../../models/projects');
+const { templates } = require('../../../../utils/responseCodes');
 
 const Containers = { ...Groups };
 
+Containers.addContainer = addModel;
+
+Containers.deleteContainer = async (teamspace, project, container, user) => {
+	try {
+		await deleteModel(teamspace, project, container, user);
```

Comment:
We've never checked if the container ID provided is indeed a container.

So as it stands you can pass a federation ID and get it deleted through this. There should be a check.

Might be easier to add it as a middleware